src/sentry/api/endpoints/organization_details.py

```diff
@@ -168,6 +168,9 @@ class OrganizationSerializer(BaseOrganizationSerializer):
     allowJoinRequests = serializers.BooleanField(required=False)
     relayPiiConfig = serializers.CharField(required=False, allow_blank=True, allow_null=True)
     apdexThreshold = serializers.IntegerField(min_value=1, required=False)
+    provider = serializers.CharField(required=False, allow_blank=True, allow_null=True)
+    provider = serializers.CharField(required=False, allow_blank=True, allow_null=True)
+    config = serializers.JSONField(required=False, allow_null=True)
```

Comment:
You should think about the naming of the keys. This is confusing because I might misunderstand it to be "config for organization".

Reply:
Changed `config` -> `providerConfig`